config/fail2ban-jail.cf

```diff
@@ -19,3 +19,16 @@
 # iptables-multiport:	block IP only on affected port
 # iptables-allports:	block IP on all ports
 #banaction = iptables-allports
+
+# Email settings
+
+#destemail = admin@example.org
+#sender = server@example.org
+#sendername = fail2ban
+#mta = sendmail
+
+# to ban & send an e-mail with whois report to the destemail.
+#action = %(action_mw)s
+
+# same as action_mw but also send relevant log lines
+#action = %(action_mwl)s
```

Comment:
I did on purpose only include those configuration options in the example config, that we customize for DMS: https://github.com/docker-mailserver/docker-mailserver/blob/master/target/fail2ban/jail.local

In my experience, you don't want an email notification on every ban 😄 Also most public whois servers are rate limited and you likely will be temporary blocked pretty fast.

If you still want to include it, then I would suggest (to keep it streamlined) , to add the original comments (from the default configuration) as well:

```
# Destination email address used solely for the interpolations in
# jail.{conf,local,d/*} configuration files.
destemail = root@localhost

# Sender email address used solely for some actions
sender = root@<fq-hostname>

# E-mail action. Since 0.8.1 Fail2Ban uses sendmail MTA for the
# mailing. Change mta configuration parameter to mail if you want to
# revert to conventional 'mail'.
mta = sendmail
```

Reply:
> In my experience, you don't want an email notification on every ban smile Also most public whois servers are rate limited and you likely will be temporary blocked pretty fast.

Thanks for letting me know, I will maybe disable whois in production then

I just wanted to document the possibility, maybe should I add this block to the docs page ?

Reply:
fyi: all available configuration options (not only mail stuff) can be easily looked up with

`docker exec mailserver cat /etc/fail2ban/jail.conf`
`docker exec mailserver cat /etc/fail2ban/fail2ban.conf`

Then you'll only need to copy/paste the neccesary parts to your custom fail2ban config and adjust them.

Reply:
Good to know, I would not even have guessed that
Will update the docs page